src/libraries/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
```

Comment:
do we need GeneratePackageOnBuild=true here since this library has changes? or does that only apply to OOB libraries?